src/main.rs

```diff
@@ -117,7 +118,10 @@ fn main() {
 
         // Construct our main `Canvas` tree.
         widget::Canvas::new().flow_right(&[
-          (ids.imgcanvas, widget::Canvas::new().color(color::CHARCOAL).border(0.0)),
+          (ids.leftarea, widget::Canvas::new().color(color::CHARCOAL).border(0.0).flow_down(&[
+              (ids.imgcanvas, widget::Canvas::new().color(color::GREY).border(0.0)),
+              (ids.footer, widget::Canvas::new().color(color::CHARCOAL).length(75.0).border(0.0)),
```

Comment:
The footer is probably too tall for just the text. It should be sized to hold 1.5 font heights or something like that.